geode-core/src/main/java/org/apache/geode/management/internal/i18n/CliStrings.java

```diff
@@ -1873,6 +1873,8 @@
       "Total primaries transferred during this rebalance";
   public static final String REBALANCE__MSG__TOTALTIME =
       "Total time (in milliseconds) for this rebalance";
+  public static final String REBALANCE__MSG__MEMBER_COUNT =
+      "Total number of members in system during rebalance";
```

Comment:
This description is potentially misleading, since the count is actually the number of members on which the rebalance was executed, which might be different from the total number of members in the system.